### ISSUE TITLE
R Fundamentals

### ISSUE THREAD

**Issue by sclappe** [UNECE/ModernStats_R#3]
The R group of the CSO started a curriculum for beginners, from our experience, I would suggest:
- Add: How to install and load packages in R?
- Shall we add a section on how to properly work in RStudio with RStudio projects?
- Shall we add a section on how to manage Directories?

**Comment by MounaBelaid** [comment]
I totally agree on that. I would also suggest to add a part about Posit Cloud and its features.

**Comment by alburycatalina** [comment]
Hi @sclappe! Thank you so much for taking the time to provide some feedback on the lesson. 

* Agreed: we've added a section on installing and using packages in the R fundamentals episode!
* There is now a section that will serve as an introduction to Rstudio. It will include a our of the IDE and an explanation of projects and directories.
